packages/coinstac-api-server/test/data/multi-shot-schema.json

```diff
@@ -33,14 +34,20 @@
         "max": 1,
         "min": 0,
         "step": 0.05,
-        "type": "number"
+        "type": "number",
+        "source": "owner"
       },
       "covariates":
       {
         "defaultValue": "[{'name': 'Is Control?','type': 'boolean'}]",
         "label": "Covariates",
         "type": "array",
         "items": ["boolean", "number"]
+      },
+      "data": {
```

Comment:
Would type be `files` here? Then the schema would let you put on a extension to test (when we do that)

Reply:
I think another field would be better to use for that. Right now, the `array` type informs the UI how to display the input field. And since the actual data structure is an array, I think it fits well.